include-build/roborazzi-gradle-plugin/src/main/java/io/github/takahirom/roborazzi/RoborazziPlugin.kt

```diff
@@ -452,6 +451,14 @@ abstract class RoborazziPlugin : Plugin<Project> {
   private fun String.capitalizeUS() =
     replaceFirstChar { if (it.isLowerCase()) it.titlecase(Locale.US) else it.toString() }
 
+  private fun FileCollection.generateHash(): String {
```

Comment:
For the `sample-android` project this seems to perform well (takes 30-40ms to find the final hash) -- I haven't tried this with a module with a large amount of tests though.